### ISSUE TITLE
Typescript compilation fails after deploy

### ISSUE THREAD

**Issue by cameronk** [graphcool/prisma#1973]
# Bug Report

**Current behavior**
I'm on Prisma 1.2.8.
When I ran `prisma deploy` to a local cluster, the `prisma.ts` and `prisma.graphql` files were updated in my `src/generated` directory (as expected). However, `prisma.ts` contains this autogenerated line:

```
"""
The `Long` scalar type represents non-fractional signed whole numeric values.
Long can represent values between -(2^63) and 2^63 - 1.
"""
scalar Long
```

This line was added right above the definition for my mutations:

```

"""
The `Long` scalar type represents non-fractional signed whole numeric values.
Long can represent values between -(2^63) and 2^63 - 1.
"""
scalar Long

type Mutation {
  createUser(data: UserCreateInput!): User!
....
```

This breaks Typescript compilation, because the backticks around "Long" interfere with the ones wrapping the entire statement:
```
const typeDefs = `....`;
```

**Reproduction**
- Update your Prisma client to 1.2.8
- Deploy a service that uses an Int type (looks like this gets renamed on the server side to Long)
- Run `prisma deploy` to a local cluster (or any cluster)

**Expected behavior?**
Either the `scalar Long` doesn't need to be in the type definition at that position, or it shouldn't have backticks in the comment.

I solved this by manually removing them from the type definition. Everything else seems to work fine.

**Comment by akoenig** [comment]
This has been fixed via https://github.com/graphql-binding/graphql-static-binding/commit/947c15ce58ffeaad5577e3f27e2c108f5426f9fb - You shouldn't face the problem when upgrading to the latest version of `graphql-cli`.

**Comment by maxdarque** [comment]
@akoenig thank you! what version number do we need to upgrade to?

**Comment by akoenig** [comment]
You're welcome :) The fix has been shipped with `v2.15.6`, but I would upgrade to the current version which is `v2.15.7`.

**Comment by akoenig** [comment]
@cameronk Have upgraded your version as well and did that solve your problem?

**Comment by maxdarque** [comment]
Upgrading fixed this issue for me

**Comment by marktani** [comment]
Looks similar to #1992.

**Comment by akoenig** [comment]
@marktani This is definitely fixed in `graphql-cli@^2.15.6` :)

**Comment by cameronk** [comment]
@akoenig Does Prisma have a dependency for `graphql-cli`? I've updated to `prisma@1.3.5` and this error still occurs.

**Comment by akoenig** [comment]
@cameronk That is strange. Could you try the following:

  - Delete node_modules
  - Install dependencies again
  - Verify version in  node_modules/graphql-cli/package.json 

Aaaand make sure that you execute the local installed graphql-cli and not a possible globally installed one. Try executing: yarn graphql prepare

Hope that helps 🙂

**Comment by marktani** [comment]
I'll close this as it was resolved in `graphql-cli@2.15.6`. Please let me know if you are still encountering this issue.

**Comment by abelovic** [comment]
I am getting this error (although not when doing prisma deploy). I have tried all the suggestions above and here is my versions:

prisma-binding: 1.5.16
graphql-cli: 2.15.8
prisma: 1.4.2

I'm trying to generate the typeDefs (using graphql-import-loader) so that I can use with webpack/serverless. See this for more info:

https://www.graph.cool/forum/t/how-to-import-prisma-typedefs-with-webpack/2761/12

**Comment by sakhmedbayev** [comment]
I am getting this error:
```
warning command prepare both exists in plugin /Users/saidakhmedbayev/node_projects/mine/myproject/server/node_modules/graphql-cli-prepare and is shipped with the graphql-cli.
The plugin is being ignored.

/Users/saidakhmedbayev/node_projects/mine/myproject/server/node_modules/ts-node/src/index.ts:330
        throw new TSError(formatDiagnostics(diagnosticList, cwd, ts, lineOffset))
              ^
TSError: ⨯ Unable to compile TypeScript
src/resolvers/Query.ts (4,8): Parameter 'parent' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (4,16): Parameter 'args' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (4,36): Parameter 'info' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (8,10): Parameter 'parent' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (8,18): Parameter 'args' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (8,38): Parameter 'info' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (21,8): Parameter 'parent' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (21,18): Binding element 'id' implicitly has an 'any' type. (7031)
src/resolvers/Query.ts (21,38): Parameter 'info' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (25,6): Parameter 'parent' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (25,14): Parameter 'args' implicitly has an 'any' type. (7006)
src/resolvers/Query.ts (25,34): Parameter 'info' implicitly has an 'any' type. (7006)
    at getOutput (/Users/saidakhmedbayev/node_projects/mine/myproject/server/node_modules/ts-node/src/index.ts:330:15)
    at Object.compile (/Users/saidakhmedbayev/node_projects/mine/myproject/server/node_modules/ts-node/src/index.ts:518:11)
    at Module.m._compile (/Users/saidakhmedbayev/node_projects/mine/myproject/server/node_modules/ts-node/src/index.ts:403:43)
    at Module._extensions..js (module.js:671:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/Users/saidakhmedbayev/node_projects/mine/myproject/server/node_modules/ts-node/src/index.ts:406:12)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
[nodemon] app crashed - waiting for file changes before starting...
```
Here is my package.json:

```
{
  "name": "server",
  "scripts": {
    "start": "dotenv -- nodemon -e ts,graphql -x ts-node src/index.ts",
    "dev": "npm-run-all --parallel start playground",
    "debug": "dotenv -- nodemon -e ts,graphql -x ts-node --inspect src/index.ts",
    "playground": "graphql playground",
    "build": "rimraf dist && tsc"
  },
  "dependencies": {
    "bcryptjs": "^2.4.3",
    "graphql-yoga": "1.14.6",
    "jsonwebtoken": "^8.2.2",
    "lodash": "^4.17.10",
    "prisma-binding": "2.0.2"
  },
  "devDependencies": {
    "@types/bcryptjs": "2.4.1",
    "dotenv-cli": "1.4.0",
    "graphql-cli": "^2.16.1",
    "nodemon": "1.17.5",
    "npm-run-all": "4.1.3",
    "prisma": "1.8.3",
    "rimraf": "2.6.2",
    "ts-node": "6.0.5",
    "tslint": "^5.2.0",
    "tslint-config-prettier": "^1.5.0",
    "tslint-plugin-prettier": "^1.3.0",
    "typescript": "2.9.1"
  }
}
```

**Comment by marktani** [comment]
@sakhmedbayev comments in closed issues are easy to miss :slightly_smiling_face: It's usually better to open a new issue instead.

In your specific case, please chime in on this reported issue: https://github.com/prismagraphql/prisma-binding/issues/166.
